modules.d/10i18n/module-setup.sh

```diff
@@ -216,7 +217,17 @@ install() {
         done
 
         for keymap in "${!KEYMAPS[@]}"; do
-            inst_opt_decompress "${keymap}"
+            if [[ -L ${keymap} ]]; then
+                maplink=$(readlink -f "${keymap}")
+                # skip symlinked directories
+                [[ -d ${maplink} ]] && continue
+
+                inst_opt_decompress "$(readlink -f "${keymap}")"
```

Comment:
You can pass `${maplink}` to `inst_opt_decompress`, see line 221.
Other than that the patch looks good, thanks!